build.sbt

```diff
@@ -100,9 +100,10 @@ lazy val macrotaskExecutorLib =
 lazy val kindProjectorCompilerPlugin =
   "org.typelevel" % "kind-projector" % kindProjector_Version cross CrossVersion.full
 
-/** [[https://github.com/monix/minitest/]] */
-lazy val minitestLib =
-  Def.setting { "io.monix" %%% "minitest-laws" % minitest_Version }
+/** [[https://github.com/scalameta/munit]] */
+lazy val munitLib =
+  Def.setting { "org.scalameta" %%% "munit" % "1.0.0-M6" }
```

Comment:
Depending on `-M6` might be unwise, although I'm not familiar with MUnit's release cycle. Anything wrong with the latest stable version `0.7.29`?

Are there major API changes?

Reply:
I'll swap it to the 0.7.29 version and see if it breaks the current code.